app/containers/WeatherContainer/tests/action.test.js

```diff
@@ -0,0 +1,11 @@
+import { weatherContainerTypes, weatherContainerCreators } from '../reducer';
+
+describe('WeatherContainer action tests', () => {
```

Comment:
this would be a test of redux-sauce functionality, since we use `createActions` to create actions

Reply:
lets remove this test